2021/41xxx/CVE-2021-41242.json

```diff
@@ -1,18 +1,101 @@
 {
```

Comment:
'https://jira.openolat.org/browse/OO-5819 ' does not match '^(ftp|http)s?://\\S+$'

Reply:
Removed space from the end of the URL